### DISCUSSION TITLE
passing an arbitrary json

### DISCUSSION THREAD

**Discussion by nbegey** [convictional/trigger-workflow-and-wait#24] [Q&A] [answered]
Hello,

I'm trying to pass a value using inputs but can't make it work.
I've tried a lot of things, but I'm missing something.

The json should be something like {"tag": 5}
so : inputs: '{"tag": 5}'

Can you help me please ? 

Thank you !

**Comment by kvaghela** [comment, marked as answer]
Got this working. On your downstream repo setup workflow_dispatch to take inputs e.g.

on:
  workflow_dispatch:
    # Inputs the workflow accepts.
    inputs:
      testurl:
        description: 'Which URL to test with'
        default: 'http://abcd.com'
        required: true


From your invoker repo yaml using trigger-workflow-and-wait, add this tag

inputs: '{ "testurl" :"http://xyz.com" }'

**Comment by duhow** [comment]
@nbegey IIRC, GitHub allows only input values as `"string"`, so try quoting the value, despite being JSON valid.

**Comment by nbegey** [comment]
Thank you very much for your help, guys !
